### PR TITLE
istio: now run-istio-tests executes both k8s and istio

### PR DESCRIPTION
**Pull request by hunchback**
the new istio test includes both k8s (ran on a system with istio) and istio tests. to check this fix you must add a CI hook enabling to write `run skydive-istio-tests` if successful it should replace the `run skydive-k8s-tests`

skip skydive-go-fmt
skip skydive-functional-tests-backend-elasticsearch
skip skydive-functional-tests-backend-orientdb
skip skydive-scale-tests
skip skydive-cdd-overview-tests
skip skydive-unit-tests
skip skydive-compile-tests
skip skydive-ppc64le-tests

### CODE REVIEW

**Comment by lebauce** [comment]
run  skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

File: scripts/ci/install-istio.sh [hunk 1]
```diff
@@ -26,14 +28,15 @@ install_istio() {
                 exit 1
         fi
 
-        chmod a+x $ISTIO_PKG
-        sudo cp $ISTIO_PKG/bin/istioctl $TARGET_DIR/.
-        sudo mv $ISTIO_PKG $TARGET_DIR/.
+        sudo cp -R $ISTIO_PKG $TARGET_DIR
```
Comment (nplanel) [inline scripts/ci/install-istio.sh]: don't install all istio package in /usr/bin .... we don't want to put files where they not belonging .... install it properly

Comment (lebauce) [reply scripts/ci/install-istio.sh]: That's precisely the point of this PR :-) Install Istio in /opt and only put the executable in /usr/bin

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by safchain** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by safchain** [comment]
@hunchback could you have a look at this failures please. Are you able to have the test passing somewhere else ?

**Comment by hunchback** [comment]
@lebauce / @safchain - I have verified that when ISTIO is installed the K8S tests (when run together) fail while individual K8S tests run. I suggest we postpone handling this PR for 2 weeks (until progressing with some of the open k8s items)

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Comment by lebauce** [comment]
run skydive-k8s-tests

**Pull request merged**